webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -44,31 +45,35 @@
                   :disabled="postDisabled || postingNews"
                   elevation="0"
                   class="btn btn-primary"
-                  @click="postNews">
+                  @click="openDrawer">
                   {{ $t("news.composer.post") }}
                 </v-btn>
               </v-app>
             </div>
           </div>
-          <div v-show="editMode" class="newsFormRightActions">
-            <button
-              id="newsEdit"
-              :disabled="updateDisabled"
-              class="btn btn-primary"
-              @click.prevent="updateNews">
-              {{ $t("news.edit.update") }}
-            </button>
-            <button
-              id="newsUpdateAndPost"
-              :disabled="news.archived ? true: updateDisabled"
-              :class="[news.archived ? 'unauthorizedPin' : '']"
-              class="btn"
-              @click.prevent="updateAndPostNews">
-              {{ $t("news.edit.update.post") }}
-            </button>
-            <button class="btn" @click="goBack">
-              {{ $t("news.composer.btn.cancel") }}
-            </button>
+          <div v-show="editMode" class="VuetifyApp">
```

Comment:
do you need really the class VuetifyApp in each block ?
If you keep it only on the parent div, will it be sufficient ?